examples/livesync/Dockerfile

```diff
@@ -1,10 +1,9 @@
 FROM node:18
 
-ENV IS_DOCKER=true
-
 WORKDIR /app
 RUN curl -f https://get.pnpm.io/v6.16.js | node - add --global pnpm
 COPY . .
+RUN IS_DOCKER=true
```

Comment:
This isn't quite right - this makes the env available in the build stage but not at runtime.

To get it to work, you have to include `IS_DOCKER=true` in the env file. We can then remove it from here.